configmap/informed_watcher.go

```diff
@@ -125,7 +158,7 @@ func (i *InformedWatcher) registerCallbackAndStartInformer(stopCh <-chan struct{
 	i.m.Lock()
 	defer i.m.Unlock()
 	if i.started {
-		return errors.New("watcher already started")
+		return fmt.Errorf("watcher already started")
```

Comment:
this should be back to `errors.New`

Reply:
`fmt.Errorf` was created to avoid a circular import issue with `errors` in early Go versions. It's been fixed since then, but what's the point importing both ?

Reply:
you're using a formatter function while having no format, which at least will scan the whole string once. No reason to do work that's totally unnecessary.

Reply:
OK, I didn't see the argument about performance. Thanks for clarifying.

Reply:
[done](https://github.com/knative/pkg/compare/c975f8ea85cb8e268338e217d34c5c251496ffdf..0994948eed981540438e919532b1e697763abaac)